store-implementation/proxy-store/src/main/java/uk/gov/gchq/gaffer/proxystore/ProxyStore.java

```diff
@@ -141,13 +141,6 @@ public JobDetail executeJob(final OperationChain<?> operationChain, final User u
         }
     }
 
-    @Override
-    protected <O> O handleOperationChain(
-            final OperationChain<O> operationChain, final Context context)
-            throws OperationException {
-        return executeOpChainViaUrl(operationChain, context);
-    }
-
     protected <O> O executeOpChainViaUrl(
```

Comment:
This method is no longer called from anywhere. Perhaps the proxy store could just override the handler for OperationChainHandler instead.